src/views/ProxyPage.vue

```diff
@@ -115,8 +122,11 @@
 
     <div id="print-content" :class="'scale-' + config.scale">
         <template v-for="(card, index) in cards" :key="index">
-            <img v-for="n in card.quantity" :key="n" :src="card.selectedOption.url" v-show="shouldShowCard(card)">
-            <img v-for="n in card.quantity" :key="n" :src="card.selectedOption.urlBack" v-show="shouldShowCard(card, 'back')">
+            <template v-for="n in card.quantity" :key="n">
+                <img :src="card.selectedOption.url" v-if="shouldShowCard(card)">
+                <img :src="card.selectedOption.urlBack" v-if="shouldShowCard(card, 'back')">
+                <img src="card_back_highres.jpg" v-else-if="config.includeBacks">
```

Comment:
This card back has a bigger border than the cropped images used for the fronts.
Your use case of actually gluing cards is pretty different from what I normally use this for, so don't know if that's something that would be an issue (or at least an annoyance) for you?

Reply:
I see. So `mtg-proxy-site` uses "border_crop" version of the images from scryfall. Looks like ideal solution for me is:

1. Change order of the cards when printing, so I can keep folding and gluing.
2. Add option to use "large" images instead of "border_crop" that that will use images like [that](https://cards.scryfall.io/normal/front/0/0/0003aa31-d42e-4972-9fb9-086aefe29a2c.jpg?1675536851). It's even [not that hard](https://codepen.io/michaelkrukov/pen/mdZERYd) to fill the corners.
3. Add option to add card backs when needed (with two card back images – one looking like "corder_crop" and other looking like "large").

That way current use cases are not really affected while I get functionality I want. I guess adding another two options is not ideal, but I have couple ideas how to tidy interface up I can try. I can split changes into different PRs, or do you think you wouldn't accept them?